src/platform-includes/distributed-tracing/instrumentation/custom-instrumentation/javascript.mdx

```diff
@@ -0,0 +1,96 @@
+We will implement "Tracing without Performance" for Javascript Browser first,
+then revisit this page and change it to match the new official apis to use.
+
+-->
+
+<Alert title="Note" level="note">
+
+All Sentry JavaScript SDKs support distributed tracing out of the box if you add the <PlatformLink to="/performance/instrumentation/automatic-instrumentation/">BrowserTracing</PlatformLink> integration to your `Sentry.init` option. The [Next.js](/platforms/javascript/guides/nextjs/) and [SvelteKit](/platforms/javascript/guides/sveltekit/) SDKs already include this integration by default.
+
+If you are using one of the integrations of Frameworks listed above, **you do not need to do anything** for distributed tracing to work!
+
+You can stop reading this page now and continue building an amazing product.
+
+</Alert>
+
+Please read the note above. If you want to implement distributed tracing "by hand" two things need to be done:
+
+- When loading the page, extract and store incoming tracing information from HTML `meta` tags.
+- When doing outgoing requests, inject tracing information to the outgoing request.
+
+To learn more see also: <PlatformLink to="/distributed-tracing/#how-does-distributed-tracing-work-in-sentry">How Does Distributed Tracing Work in Sentry?</PlatformLink>
+
+## Extract Tracing Information From HTML `meta` Tags
+
+This code snipped assumes, that the backend that renders the HTML, injects tracing information into the HTML as `meta` tags.
+
+See for example [Inject Tracing Information Into Rendered HTML](/platforms/python/distributed-tracing/instrumentation/custom-instrumentation/#inject-tracing-information-into-rendered-html) to learn how your backend can inject this information.
+
+On `pageload` do the following:
+
+```javascript
+// Read meta tag values
+const sentryTraceMetaTagValue = getMetaContent("sentry-trace");
+const baggageMetaTagValue = getMetaContent("baggage");
+
+// Extract Sentry trace information
+const traceParentData = extractTraceparentData(sentryTraceMetaTagValue);
+
+// Create Dynamic Sampling Context
+const dynamicSamplingContext =
+  baggageHeaderToDynamicSamplingContext(baggageMetaTagValue);
+
+// Create transaction context
+const transactionContext = {
+  ...traceParentData,
+  metadata: {
+    dynamicSamplingContext: dynamicSamplingContext,
+  },
+};
+
+// Start transaction with tracing information of meta tags
+const pageLoadTransaction = startTransaction(hub, transactionContext);
+```
+
+In this example we create a new transaction, that is attached to the trace that was specified in the information in the `sentry-trace` and `baggage` HTML `meta` tags.
+
+## Inject Tracing Information to Outgoing Requests
+
+On outgoing XHR/fetch requests you have to add the two headers `sentry-trace` and `baggage` that you extracted and stored in the `pageLoadTransaction` earlier.
+
+If you are do outgoing requests in another way you can generate the tracing information like follows:
```

Comment:
```suggestion
```

?

Reply:
What I meant with "in another way" is that if you are not using HTTP as protocol to talk between your service. For example you use some proprietary protocol you invented or some other obscure protocol that has no concept of headers.

Reply:
but I guess it is fine to remove